pkg/oonimkall/webconnectivity_integration_test.go

```diff
@@ -0,0 +1,28 @@
+package oonimkall_test
+
+import (
+	"testing"
+
+	"github.com/ooni/probe-cli/v3/pkg/oonimkall"
+)
+
+func TestSessionWebConnectivity(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skip test in short mode")
+	}
+	sess, err := FakeNewSession()
```

Comment:
TODO(@bassosimone): this is not a Fake session. We should use another name, like NewSessionForTesting.